components/floatingactionbutton/appearance.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Appearance
+page_title: Floating Action Button Appearance
+description: Explore the appearance settings of the Floating Action Button for Blazor. See the available options that allow you to fully customize the look of the Floating Action Button component. 
+slug: fab-appearance
+tags: telerik,blazor,floating action button, appearance
+published: True
+position: 1
+---
+
+# Appearance Settings
+
+The Floating Action Button component features built-in appearance parameters that allow you to customize virtually every aspect of its look and feel. The example at the bottom of the page lets you experiment with the available parameters.
+
+## Rounded
+
+The `Rounded` parameter applies the `border-radius` CSS rule to the button to achieve curving of the edges. You can set it to a member of the `Telerik.Blazor.ThemeConstants.Button.Rounded` class:
+
+| Class members | Manual declarations |
+|------------|--------|
+|`Small` |`sm`|
+|`Medium`|`md`|
+|`Large`|`lg`|
+|`Full`|`full`(default)|
```

Comment:
```suggestion
|`Full`|`full` (default)|
```

Reply:
I recommend adding a space for the sake of better visualization. Here's how it renders with no space.

![image](https://github.com/user-attachments/assets/c0ad5160-1307-42a5-9724-82cc3b904b67)